### PR TITLE
Update tutorials to fix hyperparameters and function name change

### PR DESCRIPTION
**Pull request by anthonyhsyu**

### CODE REVIEW

File: tune_exercises/exercise_1_basics.ipynb [hunk 1]
```diff
@@ -200,8 +200,8 @@
     "\n",
     "results = tune.run(\n",
     "    tune_iris, \n",
-    "    config={\"lr\": 0.1, \"dense_1\": 1, \"dense_2\": 0.1},\n",
-    "    num_samples=1,\n",
+    "    config={\"lr\": 0.1, \"dense_1\": 1, \"dense_2\": 2},\n",
+    "    num_samples=10,\n",
```
Comment (richardliaw) [inline tune_exercises/exercise_1_basics.ipynb]:
```suggestion
    "    num_samples=2,\n",
```

**Comment by richardliaw** [comment]
Looks good; can merge if you just reduce the first example from 10 to something lower.

File: tune_exercises/exercise_1_basics.ipynb (hunk 1 above)
Comment (anthonyhsyu) [reply tune_exercises/exercise_1_basics.ipynb]: Should we likewise change the assertion below to 2 as well?

**Comment by richardliaw** [comment]
Yeah that makes sense.

**Pull request merged**